src/_pytest/config/__init__.py

```diff
@@ -313,13 +333,11 @@ def __init__(self) -> None:
 
         super().__init__("pytest")
         # The objects are module objects, only used generically.
-        self._conftest_plugins = set()  # type: Set[object]
+        self._conftest_plugins = set()  # type: Set[types.ModuleType]
 
-        # state related to local conftest plugins
-        # Maps a py.path.local to a list of module objects.
-        self._dirpath2confmods = {}  # type: Dict[Any, List[object]]
-        # Maps a py.path.local to a module object.
-        self._conftestpath2mod = {}  # type: Dict[Any, object]
+        # State related to local conftest plugins.
+        self._dirpath2confmods = {}  # type: Dict[py.path.local, List[types.ModuleType]]
+        self._conftestpath2mod = {}  # type: Dict[Path, types.ModuleType]
```

Comment:
Actually stores `Path` and not what the previous comment said.

I removed the comments since they are redundant with the types.